.devcontainer/devcontainer.json

```diff
@@ -38,6 +47,50 @@
       "!include_dir_list scalar",
       "!include_dir_merge_list scalar",
       "!include_dir_merge_named scalar"
+    ],
+    // python common
+    "[python]": {
+      "editor.defaultFormatter": "ms-python.python",
+      "editor.codeActionsOnSave": {
+        "source.organizeImports": true
+      }
+    },
+    "python.linting.enabled": true,
+    "python.pythonPath": "/usr/local/bin/python",
+    // python linting
+    "python.linting.banditEnabled": true,
+    "python.linting.banditArgs": [
+      "--format=custom",
+      "--configfile=tests/bandit.yaml"
+    ],
+    "python.linting.flake8Enabled": true,
+    "python.linting.flake8Args": ["--config=./setup.cfg"],
+    "python.linting.mypyEnabled": true,
+    "python.linting.mypyArgs": [
+      "--follow-imports=silent",
+      "--ignore-missing-imports",
+      "--show-column-numbers",
+      "--no-pretty",
+      "--config-file=./mypy.ini"
+    ],
+    "python.linting.pylintEnabled": false,
+    "python.linting.pylamaEnabled": false,
+    "python.formatting.provider": "black",
+    "python.formatting.blackArgs": [
+      "--config=./pyproject.toml",
+      "--line-length=88"
+    ],
+    "isort.args": ["--settings-path=./pyproject.toml"],
+    "pylint.args": ["--rcfile=./pyproject.toml"],
+    // python tests
+    "python.testing.promptToConfigure": false,
+    // disabled by default due to extra needed requirements
```

Comment:
Which ones? I use it...

It is really helpful running tests with the debugger attached.

Reply:
This was already set to false in the settings.default.json, so I assume that was what people wanted. You have to run the "Install all Test Requirements" task for this to work (I probably could have included that in the comment).

The test discovery just fails until all of the requirements are installed, so we certainly could just have it enabled.